Eatery/Views/BRB/FavoriteTableViewCell.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  FavoriteTableViewCell.swift
+//  Eatery
+//
+//  Created by Noah Pikielny on 4/12/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class FavoriteTableViewCell: UITableViewCell {
+
+    private let nameLabel = UILabel()
+    private let servingLabel = UILabel()
+
+    private var favoriteStatus = UIImageView(image: .favoritedImage)
+    public var favorited = true {
+        didSet {
+            favoriteStatus.image = favorited ? .favoritedImage : .unfavoritedImage
+            favoriteStatus.tintColor = favorited ? .favoriteYellow : .lightGray
+        }
+    }
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        selectionStyle = .none
+
+        nameLabel.font = .preferredFont(forTextStyle: .headline)
+        nameLabel.numberOfLines = 0
+        contentView.addSubview(nameLabel)
+        nameLabel.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(20)
+            make.top.equalToSuperview().inset(12)
+        }
+
+        servingLabel.font = .preferredFont(forTextStyle: .subheadline)
+        servingLabel.textColor = .gray
+        contentView.addSubview(servingLabel)
+        servingLabel.snp.makeConstraints { make in
+            make.top.equalTo(nameLabel.snp.bottom).offset(2)
+            make.leading.equalToSuperview().inset(20)
```

Comment:
20 seems to be used a few times for .leading.  Would you be able to set some variable (for example)
``` leadPadding = 20 ``` to express this redundancy in a variable and make this a little more scalable.